src/test/java/org/heigit/ohsome/ohsomeapi/controller/DataExtractionTest.java

```diff
@@ -129,7 +130,18 @@ public void elementsBboxTest() {
     assertEquals("Polygon", featureGeom.get("type").asText());
     assertEquals(5, featureGeom.get("coordinates").get(0).size());
   }
-
+  
+  @Test
+  public void contributionTypesValueElementsBboxTest() {
```

Comment:
I'd prefer to use a more meaningful naming scheme for the tests. I've started it with the tests for `/contributions/count` as you can see for example with this test:
https://github.com/GIScience/ohsome-api/blob/a177dc3d3ccb6c74ccfaa9a117925271e3ffee76/src/test/java/org/heigit/ohsome/ohsomeapi/controller/PostControllerTest.java#L1144
When a general re-organization of the tests would be performed at some point, it would be great to apply such a scheme to all test methods. Am also open to discuss about some other standards regarding the naming of test methods.